Buildings/Controls/OBC/ASHRAE/G36/Atomic/ActiveAirflowSetpointReheatBox.mo

```diff
@@ -0,0 +1,987 @@
+within Buildings.Controls.OBC.ASHRAE.G36.Atomic;
+block ActiveAirflowSetpointReheatBox
+  "Output the active airflow setpoint for VAV reheat terminal unit"
+
+  parameter Boolean occSen = true
+    "Set to true if the zone has occupancy sensor"
+    annotation(Dialog(group="Zone sensors"));
+  parameter Boolean winSen = true
```

Comment:
I don't understand how this can be a parameter: whether the window is open or closed changes during the simulation, but parameters cannot be changed. Should this rather be an input?

Reply:
@mwetter `winSen` is a parameter for indicating if the zone has window status sensor. There is an input `uWin` to show current window status.